exams/pearson_test.py

```diff
@@ -1,32 +1,35 @@
 """
 Tests for exams.pearson module
 """
-from datetime import datetime
+from datetime import date, datetime
 from unittest import mock
 from unittest.mock import patch
 
 import io
```

Comment:
I'll adjust them then, as much as I have my own preferences, I much more don't like having islands of personal code styles dotted though code.